tests/functional/spec/features/validation/sum/value_source.spec.js

```diff
@@ -0,0 +1,142 @@
+      $(TotalAnswerPage.total()).setValue("12");
+      $(TotalAnswerPage.submit()).click();
+
+      answerBothBreakdownQuestions(["3", "3", "3", "3"], ["2", "2", "1", "1"]);
+
+      expect(browser.getUrl()).to.contain(SubmitPage.pageName);
+    });
+  });
+
+  describe("Given I completed both grouped answer validation questions and I am on the summary", () => {
+    it("When I go back from the summary and change the total, Then I must reconfirm both breakdown questions with valid answers before I can get to the summary", () => {
+      $(TotalAnswerPage.total()).setValue("12");
+      $(TotalAnswerPage.submit()).click();
+
+      answerBothBreakdownQuestions(["3", "3", "3", "3"], ["2", "2", "1", "1"]);
+
+      $(SubmitPage.totalAnswerEdit()).click();
+      $(TotalAnswerPage.total()).setValue("15");
+      $(TotalAnswerPage.submit()).click();
+
+      answerBothBreakdownQuestions(["6", "3", "3", "3"], ["3", "3", "2", "1"]);
+
+      expect(browser.getUrl()).to.contain(SubmitPage.pageName);
+    });
+  });
+
+  describe("Given I completed both grouped answer validation questions and I am on the summary", () => {
+    it("When I go back from the summary and change the total, Then I must reconfirm the breakdown question based on answer value source with valid answers before I can continue", () => {
+      $(TotalAnswerPage.total()).setValue("12");
+      $(TotalAnswerPage.submit()).click();
+
+      answerBothBreakdownQuestions(["3", "3", "3", "3"], ["2", "2", "1", "1"]);
+
+      $(SubmitPage.totalAnswerEdit()).click();
+      $(TotalAnswerPage.total()).setValue("15");
+      $(TotalAnswerPage.submit()).click();
+
+      answerAndSubmitBreakdownQuestion("0", "3", "3", "3");
+
+      expect(browser.getUrl()).to.contain(BreakdownAnswerPage.pageName);
+
+      expect($(BreakdownAnswerPage.errorNumber(1)).getText()).to.contain("Enter answers that add up to 15");
+
+      answerBothBreakdownQuestions(["5", "4", "4", "2"], ["3", "3", "2", "1"]);
+
+      expect(browser.getUrl()).to.contain(SubmitPage.pageName);
+    });
+  });
+
+  describe("Given I completed both grouped answer validation questions and I am on the summary", () => {
+    it("When I go back from the summary and change the total, Then I must reconfirm the breakdown question based on answer value source with valid answers before I can continue", () => {
+      $(TotalAnswerPage.total()).setValue("12");
+      $(TotalAnswerPage.submit()).click();
+
+      answerBothBreakdownQuestions(["3", "3", "3", "3"], ["2", "2", "1", "1"]);
+
+      $(SubmitPage.totalAnswerEdit()).click();
+      $(TotalAnswerPage.total()).setValue("15");
+      $(TotalAnswerPage.submit()).click();
+
+      answerBothBreakdownQuestions(["6", "3", "3", "3"], ["1", "1", "1", "1"]);
+
+      expect(browser.getUrl()).to.contain(SecondBreakdownAnswerPage.pageName);
+
+      expect($(SecondBreakdownAnswerPage.errorNumber(1)).getText()).to.contain("Enter answers that add up to 9");
+
```

Comment:
Why are we changing the value source answer here?. This is just about calculated summary value and it's breakdown.

Reply:
But you are changing the first answer value which this test case shouldn't care about as that you test in the one above. This should just be when you edit the breakdown that make up the calculated summary total, then the second breakdown has validator errors?.

Reply:
This test is also about changing the value of the total answer (first answer value), I just do what is in the description, if we don't want to change the total answer then this test should have different description or needs to be removed if we don't want to test that.

Reply:
You already test changing answer value source in the previous spec. L76.

Reply:
The test is still edit the main answer instead of just the breakdown.